### PR TITLE
parent dachboard  layout

### PR DESCRIPTION
**Pull request by IrynaKolh**

### CODE REVIEW

File: src/models/interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+interface StudentTuterInfo {
```
Comment (oxangyal) [inline src/models/interfaces.ts]: a little typo Tutor:)

File: src/theme.ts
Comment (oxangyal) [inline src/theme.ts:1]: Lets create utilis folder and put theme.ts there?

File: src/models/interfaces.ts (hunk 1 above)
Comment (IrynaKolh) [reply src/models/interfaces.ts]: fixed

File: src/theme.ts
Comment (IrynaKolh) [reply src/theme.ts:1]: done

**Review by ElenaDu — APPROVED** [review]
Looks good!

**Review by katsiarynalashcheuskaya — APPROVED** [review]
Great job, Iryna! 
Everything looks good to me.

**Pull request merged**